evm/v0.5/testv2/Coordinator.test.ts

```diff
@@ -0,0 +1,909 @@
+import * as h from '../src/helpers'
+import { ethers } from 'ethers'
+import { LinkTokenFactory } from '../src/generated/LinkTokenFactory'
+import { CoordinatorFactory } from '../src/generated/CoordinatorFactory'
+import { EmptyAggregatorFactory } from '../src/generated/EmptyAggregatorFactory'
+import { MeanAggregatorFactory } from '../src/generated/MeanAggregatorFactory'
+import { GetterSetterFactory } from '../src/generated/GetterSetterFactory'
+import { MaliciousRequesterFactory } from '../src/generated/MaliciousRequesterFactory'
+import { MaliciousConsumerFactory } from '../src/generated/MaliciousConsumerFactory'
+import { Instance } from '../src/contract'
+import { makeTestProvider } from '../src/provider'
+import { assert } from 'chai'
+import { assertBigNum } from '../src/matchers'
+import { BigNumberish } from 'ethers/utils'
+
+const provider = makeTestProvider()
+
+const linkTokenFactory = new LinkTokenFactory()
+const coordinatorFactory = new CoordinatorFactory()
+const emptyAggregatorFactory = new EmptyAggregatorFactory()
+const meanAggregatorFactory = new MeanAggregatorFactory()
+const getterSetterFactory = new GetterSetterFactory()
+const maliciousRequesterFactory = new MaliciousRequesterFactory()
+const maliciousConsumerFactory = new MaliciousConsumerFactory()
+
+let roles: h.Roles
+
+const fs = (name: string) =>
+  h.getFunctionSignature(emptyAggregatorFactory, name)
```

Comment:
this helper function uses ethersjs's `interface.functions.<functionName>.sighash` property... my impression was that we were generally wrapping ethersjs utilities with our own helper functions. I think that at least keeps the test files reliant on one helpers module, rather than using both `helpers` and `ethers.utils`.